hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/config/MessagePreviewProperties.java

```diff
@@ -0,0 +1,47 @@
+package pl.allegro.tech.hermes.frontend.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "frontend.message.preview")
+public class MessagePreviewProperties {
+
+    private boolean enabled = false;
+
+    private int maxSizeKb = 10;
+
+    private int size = 3;
+
+    private int logPersistPeriodSeconds = 30;
```

Comment:
[minor] maybe we can make it as `Duration` ? It would be defined as `frontend.message.preview.logPersistPersionSeconds: 30s` in application.yaml, it would be easy to convert it to seconds in the `TopicMetadataLoadingJob` class too

Reply:
Good point, but it also applies to already merged changes, I will add a comment there and correct it all together in one place